src/settings/ManageAuthoritySourceFiles/ManageAuthoritySourceFiles.test.js

```diff
@@ -173,4 +173,23 @@ describe('Given Settings', () => {
       });
     });
   });
+
+  describe('when validating an item', () => {
+    beforeEach(async () => {
+      const { getByText } = renderManageAuthoritySourceFiles();
+
+      await act(() => userEvent.click(getByText('stripes-core.button.new')));
+    });
+
+    describe('when a "Prefix" field is not alphabetical', () => {
```

Comment:
Please also add this test case to `getValidators.test.js` - that file contains all other fields' validation tests

Reply:
I added tests in the `ManageAuthoritySourceFiles.test.js` instead of `getValidators.test.js` since if the `validate` function doesn't work correctly the tests will fail but in `getValidators.test.js` they will pass. It is more robust having them here.

Reply:
I agree that we need to have some tests in this file to test integration, and in this case this test can be more generally describled like "when trying to save an invalid file an error should be shown"
`getValidators.test.js` describes validation more precisely - which fields should be validated and what the validation rules are